patches/@changesets+assemble-release-plan+3.0.1.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@changesets/assemble-release-plan/dist/assemble-release-plan.cjs.dev.js b/node_modules/@changesets/assemble-release-plan/dist/assemble-release-plan.cjs.dev.js
+index de55d40..afd5e5d 100644
+--- a/node_modules/@changesets/assemble-release-plan/dist/assemble-release-plan.cjs.dev.js
++++ b/node_modules/@changesets/assemble-release-plan/dist/assemble-release-plan.cjs.dev.js
+@@ -325,7 +325,7 @@ function _defineProperty(obj, key, value) { if (key in obj) { Object.definePrope
+ 
+ function getPreVersion(version) {
+   let parsed = semver.parse(version);
+-  let preVersion = parsed.prerelease[1] === undefined ? -1 : parsed.prerelease[1];
++  let preVersion = -1;
```

Comment:
yes, i know it's excessive, but couldnt help myself 🤷‍♂️

Reply:
I've confirmed that it works locally OK. We'll review exact published versions when bot prepares a versioning PR